src/auth/GitHubApolloProvider.tsx

```diff
@@ -56,7 +58,22 @@ const GitHubApolloProvider: React.FC = ({ children }) => {
   }
 
   if (linkErrorMessage) {
-    return <div>{linkErrorMessage}</div>;
+    return (
+      <Box sx={{ flexGrow: 1 }}>
+        <Grid container>
+          <Grid xs={4}></Grid>
+          <Grid xs={4} item style={{ textAlign: 'center' }}>
+            <ErrorTwoToneIcon sx={{ fontSize: 250 }} color="error" />
+          </Grid>
+          <Grid xs={4}></Grid>
+          <Grid xs={4}></Grid>
+          <Grid xs={4} style={{ textAlign: 'center' }} item>
+            <Typography variant="h5">{linkErrorMessage}</Typography>
```

Comment:
I wonder if it will be helpful to add a note if the hostname is not `api.github.com`. 

`If your company required VPN access to GitHub Enterprise, check to see if you are on the network`

Reply:
Yeah, I think in general we need to make the error messages better - I was assuming we'd do that more holistically but can add it here.